### DISCUSSION TITLE
[DELIVER] Auto upload for Privacy Options

### DISCUSSION THREAD

**Discussion by stherold** [fastlane/fastlane#17597] [Ideas] [answered]
Apple announced (https://developer.apple.com/news/?id=vlj9jty9) that we all need to answer privacy related questions soon. I thought it might make sense to add them to the options deliver can automatically send to App Store Connect so we don't have to specify them all over and can version them in git together with our app version.

Find them on App Store Connect under one of your apps -> `App Privacy`

**Comment by rogerluan** [comment, marked as answer]
Hey there @stherold 👋 

Thanks for bringing this up! We're already discussing this topic here: https://github.com/fastlane/fastlane/discussions/17467 🤗
